{{cookiecutter.repository_name}}/README.md

```diff
@@ -5,7 +5,8 @@
 
```

Comment:
I added some extras to the README in my PRs, maybe we want to include (some of) that? see e.g. https://github.com/globality-corp/web-annotation-extractor/pull/3/files#diff-04c6e90faac2675aa89e2176d2eec7d8